operator_learning_modules/zpk/zpk_operator_learning.py

```diff
@@ -102,13 +105,15 @@ def get_batch_probs(data):
 
         # Update all learned_operators
         if is_updated:
+            self._planning_operators.clear()
             self._learned_operators.clear()
             for ndr_set in self._ndrs.values():
                 for i, ndr in enumerate(ndr_set):
                     operator = ndr.determinize(name_suffix=i)
                     # No point in adding an empty effect or noisy effect operator
                     if len(operator.effects.literals) == 0 or NOISE_OUTCOME in operator.effects.literals:
                         continue
+                    self._planning_operators.add(operator)
```

Comment:
So the idea for standard ZPK is just that the learning and planning ops are always the same?

Reply:
yes